modules/aggregation-layers/src/aggregation-layer-v9/cpu-aggregator/aggregate.ts

```diff
@@ -67,7 +70,7 @@ export function aggregate({
   /** Given the index of a data point, returns its value */
   getValue: (index: number) => number;
   /** Method used to reduce a list of values to one number */
-  operation: AggregationOperation;
+  operation: AggregationOperation | AggregationFunc;
```

Comment:
Nit: Could make types and code a little cleaner (remove overloading / dynamic type checks) by adding `AggregationOperation: 'CUSTOM'` and supplying the func in a separate `aggregationFunction`... prop?

Perhaps the builtin modes just set a pre-defined aggregationFunction which would read quite easily?